BuildToolsCliVersion.txt

```diff
@@ -0,0 +1 @@
+1.0.0-preview2-002733
```

Comment:
We cannot use stage0 without upgrading all of build tools to use the RTM framework; I spoke with @weshaggard about this and he has plans to change build tools to use any CLI / framework, but those are not yet in the pipeline. He suggested I bring down the CLI that build tools needed instead and then we can work towards the right long-term solution going forward